wasp/draw565.py

```diff
@@ -300,20 +300,23 @@ def string(self, s, x, y, width=None, right=None):
                       fill the full width with the background colour (to ensure
                       that if we update one string with a narrower one there is no
                       need to "undraw" it).
-        :param right: Set the justification mode.  The default is to centre, if
-                      set to True then right justify, if False then left justify.
+        :param right: Deprecated: use justify=1 instead
```

Comment:
We don't have to nuke the documentation on how right works. Better to keep the original docs and add that is deprecated and replaced by justify.